galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbValidatorImpl.java

```diff
@@ -298,14 +300,14 @@ private boolean checkViewString(JsonObject view, String field, String value) {
     }
 
     private void checkVersion(String version, int minVersion, int minRelease, int minModification)
-            throws CouchdbRasException {
+            throws CouchdbException {
         String minVRM = minVersion + "." + minRelease + "." + minModification;
 
         Pattern vrm = Pattern.compile("^(\\d+)\\.(\\d+)\\.(\\d+)$");
```

Comment:
Whenever I see a regex, I want to see it tested massively with unit tests to make sure it does what you think. So having that regex as a constant might help that. 

Pre-compiling it only once would also save us compiling it multiple times = quicker.